conference_scheduler/resources.py

```diff
@@ -61,3 +61,8 @@ class Shape(NamedTuple):
     """Represents the shape of a 2 dimensional array of events and slots"""
     events: int
     slots: int
+
+
+class Constraint(NamedTuple):
+    label: str
+    expression: bool
```

Comment:
Small Technicality. Could we change `expression` to be `condition`? Technically an expression would just be one side of the constraint.